web/themes/custom/pineapple/sass/components/_card.scss

```diff
@@ -1,4 +1,4 @@
-.article--teaser {
+.card-container {
```

Comment:
Remove the 75% width on desktop from this file, that styling is dependent on the page the card is on and not the card itself so it shouldn't be within this file.

Reply:
done